image/image.go

```diff
@@ -301,6 +301,13 @@ func createBundle(w walker, m *manifest, dest, rootfs string) error {
 			if err2 := os.MkdirAll(dest, 0755); err2 != nil {
 				return err2
 			}
+			defer func() {
+				if err != nil {
+					if err3 := os.RemoveAll(dest); err3 != nil {
```

Comment:
With this defer, it may be good to use named returns.

Reply:
Fine, updated